packages/react/src/tools/array.js

```diff
@@ -29,3 +29,14 @@ export function equals(arr1, arr2) {
   }
   return true;
 }
+
+/**
+ * helper to define valid column widths for prop checking
+ *
+ * @param {object} params - inclusive edge markers
+ * @param {number} params.start
+ * @param {number} params.end
+ * @returns {number[]} arr with each item having val of its index
+ */
+export const range = ({ start = 0, end }) =>
```

Comment:
I bet we could just inline this in the module instead of pulling it out 👍

Reply:
179e50b